docroot/index.php

```diff
@@ -9,11 +9,12 @@
  */
 
 use Drupal\Core\DrupalKernel;
+use Drupal\prod_no_redirect\ProdNoRedirectDrupalKernel;
 use Symfony\Component\HttpFoundation\Request;
 
 $autoloader = require_once 'autoload.php';
 
-$kernel = new DrupalKernel('prod', $autoloader);
+$kernel = new ProdNoRedirectDrupalKernel('prod', $autoloader);
```

Comment:
@weitzman When I run composer install locally, this change gets overwritten and reverted back to using `DrupalKernel`

Reply:
I added an instruction for drupal-scaffold composer plugin so not overwrite.